config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Define the root path route
+  root "users#index"
 
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
+  # Users routes
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index]
+  end
 
-  # Defines the root path route ("/")
-  # root "posts#index"
+  # Posts routes
+  resources :posts, only: [:show]
```

Comment:
- [ ] Great job with the routes @Eccb7, but currently I can see you have a route for the post#show method outside of the user and post ID. The `post#show` method belongs to a `users/user_id/posts/post_id`, hence it is better to have it inside the `resources :users, only: [:index, :show] do ... end ` block. This makes your route more accurate and less verbose.